### PR TITLE
fix: Provide an button that allows the user to open the recipe app fr…

### LINKED ISSUE #468: Provide an button that allows the user to open the recipe app from the recipe screen
As a user, after starting a recipe, I have no way to understand what I can do next - which should be opening the application.

Screenshot 2024-03-08 at 11 46 16

We would like to add a clear button that indicates to the end-user they can open the application in a browser.
I'd suggest we add a button "Open AI App" as we have "Start AI App".

### PR DESCRIPTION
**Pull request by jeffmaury**
…om the recipe screen

Fixes #468

### What does this PR do?

Add a new Open action for a recipe or an AI app. If there is a single port, then no quickpick and browser is open without user interaction

### Screenshot / video of UI

open

### What issues does this PR fix or reference?

#468

### How to test this PR?

1. Start a recipe
2. From the recipe screen click on the Open icon
3. From the AI apps page you should see the same icon

### CODE REVIEW

File: packages/backend/src/studio-api-impl.ts [hunk 1]
```diff
@@ -236,6 +240,50 @@ export class StudioApiImpl implements StudioAPI {
       });
   }
 
+  async requestOpenApplication(recipeId: string, modelId: string): Promise<void> {
+    const recipe = this.catalogManager.getRecipeById(recipeId);
```
Comment (axel7083) [inline packages/backend/src/studio-api-impl.ts:244]: We might want to keep the implementation simple and move the logic outside ?

**Review by vrothberg — COMMENTED** [review]
Lovely improvement, thanks!

**Comment by slemeur** [comment]
👍

**Review by vrothberg — COMMENTED** [review]
Is this PR ready to get in?

**Comment by jeffmaury** [comment]
> Is this PR ready to get in?

Needs code refactoring but feature wise it is ready

**Pull request merged**